### PR TITLE
feat(osu!pppy): add buttons and multiLanguage

### PR DESCRIPTION
**Pull request by StrikerFRFX**
## Description 

Fixed a few bugs due to site internal updates 
Added localized strings

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

Provided if requested due to volume of screenshots

</details>

### CODE REVIEW

**Review by theusaf — CHANGES_REQUESTED** [review]
It looks like your branch is very old. `metadata.json` files are no longer tracked in `dist`, but in the presence's root directory.

**Comment by EncryptedDev** [comment]
Rebased your branch, still outdated lol

File: websites/O/osu!ppy/metadata.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
 		"es": "El ritmo está a solo un *clic* de distancia! Con Ouendan/EBA, Taiko y modos de juego originales, así como un editor de niveles totalmente funcional.",
 		"nl": "Ritme is slechts een *klik* verwijderd!  Met Ouendan/EBA, Taiko en originele spelmodi, en zelfs een volledig functionele level editor.",
 		"fi": "Rytmi on vain klikkauksen päässä! Varustettuna Ouendan/EBA ja Taikosta tutuilla sekä alkuperäisillä pelimuodoilla ja täysin toiminnallisella tasoeditorilla.",
+		"ga_IE": "Níl i rithim ach *cliceáil* ! Le Ouedan / EBA, Taiko agus modhanna bunaidh gameplay, chomh maith le eagarthóir leibhéal ag feidhmiú go hiomlán.",
```
Comment (EncryptedDev) [inline websites/O/osu!ppy/metadata.json:26]: @callumok2004

**Review by EncryptedDev — APPROVED** [review]
LGTM, PreMiD/Localization#7189 needs merging too

**Review by DarkVillager — APPROVED** [review]

**Comment by EncryptedDev** [comment]
Conflicts should be resolved

**Comment by StrikerFRFX** [comment]
> Conflicts should be resolved

what conflicts

**Review by Bas950 — APPROVED** [review]

**Comment by Bas950** [comment]
Bruh before I started reviewing 0 conflicts, after review: 
image

**Comment by StrikerFRFX** [comment]
when

**Comment by Timeraa** [comment]
Never received any updates.

**Pull request closed without merging**